### PR TITLE
Re-add tutorial for exist_data_new_model

### PR DESCRIPTION
**Pull request by v-qjqs**
This PR re-add tutorial for 3_exist_data_new_model.md

### CODE REVIEW

File: docs/3_exist_data_new_model.md [hunk 1]
```diff
@@ -1,12 +1,13 @@
-# 3: Inference and training with existing models and standard datasets
+# 3: Train with customized models and standard datasets
 
-In this note, you will know how to inference, test, and train predefined models with your own settings for standard datasets. We use the cityscapes dataset to train a COCO pretrained Cascade Mask R-CNN model as an example to describe the whole process.
+In this note, you will know how to inference, test, and train your own customized models under standard datasets. We use the cityscapes dataset to train a customized Cascade Mask R-CNN R50 model as an example to describe the whole process, which use [`AugFPN`](https://github.com/Gus-Guo/AugFPN) to replace defalut `FPN` as neck, and add `Rotate` or `Translate` as training-time auto augmentation.
```
Comment (AemikaChow) [inline docs/3_exist_data_new_model.md:3]: 先train 再test？Here we use the Cityscapes... to demonstrate the whole process.

Comment (AemikaChow) [reply docs/3_exist_data_new_model.md:3]: 这一段感觉英文可以再polish一下下

File: docs/3_exist_data_new_model.md [hunk 2]
```diff
@@ -50,9 +51,67 @@ python tools/convert_datasets/cityscapes.py ./data/cityscapes --nproc 8 --out-di
 Currently the config files in `cityscapes` use COCO pre-trained weights to initialize.
 You could download the pre-trained models in advance if network is unavailable or slow, otherwise it would cause errors at the beginning of training.
 
+## Prepare your own customized model
+The second step is to use your own module or training setting. Assume that we want to implement a new neck called `AugFPN` to replace with the default `FPN` under the existing detector Cascade Mask R-CNN R50. The following implements a new neck `AugFPN` under MMDetection.
```
Comment (AemikaChow) [inline docs/3_exist_data_new_model.md:55]: The following implements a new neck... 这句话也不太通顺？

File: docs/3_exist_data_new_model.md [hunk 3]
```diff
@@ -1,12 +1,13 @@
-# 3: Inference and training with existing models and standard datasets
+# 3: Train with customized models and standard datasets
 
-In this note, you will know how to inference, test, and train predefined models with your own settings for standard datasets. We use the cityscapes dataset to train a COCO pretrained Cascade Mask R-CNN model as an example to describe the whole process.
+In this note, you will know how to train, test and inference your own customized models under standard datasets. We use the cityscapes dataset to train a customized Cascade Mask R-CNN R50 model as an example to demonstrate the whole process, which use [`AugFPN`](https://github.com/Gus-Guo/AugFPN) to replace defalut `FPN` as neck, and add `Rotate` or `Translate` as training-time auto augmentation.
```
Comment (AemikaChow) [inline docs/3_exist_data_new_model.md:3]: w.r.t. using ....

File: docs/3_exist_data_new_model.md [hunk 4]
```diff
@@ -1,12 +1,13 @@
...
 
 The basic steps are as below:
 
 1. Prepare the standard dataset
-2. Prepare a config
-3. Train, test, inference models on the standard dataset.
+2. Prepare your own customized model
+3. Prepare a config
+4. Train, test, inference models on the standard dataset.
```
Comment (AemikaChow) [inline docs/3_exist_data_new_model.md:10]: and inference

File: docs/3_exist_data_new_model.md [hunk 5]
```diff
@@ -50,9 +51,67 @@ python tools/convert_datasets/cityscapes.py ./data/cityscapes --nproc 8 --out-di
 Currently the config files in `cityscapes` use COCO pre-trained weights to initialize.
 You could download the pre-trained models in advance if network is unavailable or slow, otherwise it would cause errors at the beginning of training.
 
+## Prepare your own customized model
+
+The second step is to use your own module or training setting. Assume that we want to implement a new neck called `AugFPN` to replace with the default `FPN` under the existing detector Cascade Mask R-CNN R50. The following implements`AugFPN` under MMDetection.
+
+### 1. Define a new neck (e.g. AugFPN)
+
+Create a new file `mmdet/models/necks/augfpn.py`.
```
Comment (AemikaChow) [inline docs/3_exist_data_new_model.md:60]: firstly

**Review by AemikaChow — COMMENTED** [review]
minor revision

**Review by AemikaChow — APPROVED** [review]

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**